doc_conf/conf.py

```diff
@@ -90,7 +90,7 @@
 # built documents.
 #
 # The short X.Y version.
-from hidimstat._version import __version__  # noqa
+from hidimstat import __version__  # noqa
```

Comment:
```suggestion
from hidimstat import __version__
```
I guess your change fixes the unresolved import

Reply:
yes